generators/client/templates/angular/src/main/webapp/app/shared/alert/alert.component.ts.ejs

```diff
@@ -30,24 +30,25 @@ import { JhiAlertService } from 'ng-jhipster';
             </div>
         </div>`
 })
-export class <%=jhiPrefixCapitalized%>AlertComponent implements OnInit, OnDestroy {
-    alerts: any[];
+export class AlertComponent implements OnInit, OnDestroy {
+    alerts: JhiAlert[] = [];
 
     constructor(private alertService: JhiAlertService) { }
 
-    ngOnInit() {
+    ngOnInit(): void {
         this.alerts = this.alertService.get();
     }
 
-    setClasses(alert) {
-        return {
-            'jhi-toast': alert.toast,
-            [alert.position]: true
-        };
-    }
-
-    ngOnDestroy() {
-        this.alerts = [];
+    setClasses(alert: JhiAlert): { [key: string]: boolean } {
+        const classes = { 'jhi-toast': !!alert.toast };
```

Comment:
Instead of `!!alert.toast` I think you can write `Boolean(alert.toast)` , I find it more readable WDYT ?

Reply:
I agree, made change, thanks for pointing out!